tests/Stormpath/Tests/OtherTest.php

```diff
@@ -0,0 +1,19 @@
+<?php namespace Stormpath\Tests;
+
+
+
+use Stormpath\Util\UserAgent;
+
+class OtherTest extends BaseTest {
```

Comment:
Please use the specific class's name for this test. For example: `UserAgentBuilderTest`.

Also, please create a Stormpath/Tests/Utils folder and move this class over there.